mkdocs.yml

```diff
@@ -4,6 +4,9 @@ repo_url: https://github.com/mrparkers/terraform-provider-keycloak
 nav:
 - Getting Started: index.md
 - Data Sources:
+  - keycloak_openid_client: data_sources/keycloak_openid_client.md
+  - keycloak_openid_client_authorization_policy: data_sources/keycloak_openid_client_authorization_policy.md
```

Comment:
So this is awkward, but there actually aren't doc files for these data sources you added.  I'd appreciate a PR to add these if you're interested :smile:

If not, could you remove these dead links?

Reply:
Whops! My mistake, I planned on adding those documents but I read that you were working on it so I deleted the files, but I forgot to delete the links :D

I'll file a separate PR soon, for now I'm deleting the references.